lib/rubocop/remote_config.rb

```diff
@@ -40,11 +43,13 @@ def request(uri = @uri, limit = 10, &block)
       end
 
       handle_response(http.request(request), limit, &block)
+    rescue SocketError
+      handle_response(NetworkInaccessible.new, limit, &block)
```

Comment:
Do you really need the new `NetworkInaccessible` class, or could you just use the `SocketError` object?

Reply:
Nah, it's not necessary. It provides some abstraction if there are more cases to deal with than just `SocketError`, but I haven't seen others yet.